main.nf

```diff
@@ -17,11 +17,10 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/diaproteomics --input 'sample_sheet.tsv' --input_spectral_library 'library_sheet.tsv' --irts 'irt_sheet.tsv' --swath_windows 'swath_windows.txt' -profile standard,docker
+    nextflow run nf-core/diaproteomics --input 'sample_sheet.tsv' --input_spectral_library 'library_sheet.tsv' --irts 'irt_sheet.tsv' -profile standard,docker
 
     Mandatory arguments:
-      --input                           Path to input DIA raw/mzML data sheet (must be surrounded with quotes)
-      --swath_windows                   Path to swath_windows.txt file, containing swath window mz ranges
```

Comment:
out of interest: why has this been removed (as well as prec_charge flag)?

Reply:
So the input flat remains, but I removed the swath_windows since I realized that the OpenSwathWorkflow is able to read them from the DIA raw file.
The prec_charge flag was a remaining code line from a previous version but has never been used and is actually not an input for the OpenSwathWorkflow.